scripts/select_appointments.sql

```diff
@@ -1 +1,14 @@
 -- retrieve appointment information
+SELECT p.given_name, p.surname, a.time
+FROM appointments a
+INNER JOIN patients p
+  ON a.patient_id = p.id
+WHERE a.doctor_id = 6
+ORDER BY p.surname;
+
+SELECT d.given_name, d.surname, a.time
+FROM appointments a
+INNER JOIN doctors d
+  ON a.doctor_id = d.id
+WHERE a.patient_id = 8
+ORDER BY d.surname;
```

Comment:
Your solution does provide the result we want, but your implementation doesn't follow what the question asked. You are knowledgable about the id number of every doctor and patient because you have direct access to the data set. But the question asks for appointments of "Dr. Hubbard," which is what your user would likely search with. Therefore, the `WHERE` clause should query doctors by name. 

Here's 1 possible solution:
```sql
SELECT p.surname, a.date, a.time
  FROM patients p
  INNER JOIN appointments a ON a.patient_id = p.id
  INNER JOIN doctors d ON a.doctor_id = d.id
    WHERE d.surname = 'Hubbard';

SELECT d.surname, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a ON a.doctor_id = d.id
  INNER JOIN patients p ON a.patient_id = p.id
    WHERE p.surname = 'Wilcox';
```